### ISSUE TITLE
About restore options.

### ISSUE THREAD

**Issue by zolegg** [BlobBackup/BlobBackup#54]
Restore a single file or directory without saving the full path structure. Is this possible?

**Comment by bimbashrestha** [comment]
Sure! A single file restore should be possible. What do you think the right way to do this is? Maybe if a a user tries to restore only a single file, just default to resorting it without the full path?

**Comment by zolegg** [comment]
Yes, I think default to restoring without the full path, and optional with full path will be nice.
